ReSwift/CoreTypes/Action.swift

```diff
@@ -52,7 +52,7 @@ extension StandardAction: Coding {
 
     public init?(dictionary: [String: AnyObject?]) {
         guard let type = dictionary[typeKey] as? String,
-          isTypedAction = dictionary[isTypedActionKey] as? Bool else { return nil }
+          let isTypedAction = dictionary[isTypedActionKey] as? Bool else { return nil }
```

Comment:
This warning will be fixed with #132 - it seemed better to address swiftlint related changes in a different chunk.